interactors/nonceHandlerV3/addressNonceHandler.go

```diff
@@ -58,14 +59,12 @@ func NewAddressNonceHandlerV3(proxy interactors.Proxy, address sdkCore.AddressHa
 // ApplyNonceAndGasPrice will apply the computed nonce to the given FrontendTransaction
 func (anh *addressNonceHandler) ApplyNonceAndGasPrice(ctx context.Context, txs ...*transaction.FrontendTransaction) error {
 	for _, tx := range txs {
-		//anh.mut.Lock()
-		nonce, err := anh.fetchNonce(ctx)
+		nonce, err := anh.computeNonce(ctx)
 		if err != nil {
 			//anh.mut.Unlock()
```

Comment:
comment remained here

Reply:
I am pretty sure it's not there any longer. Might be github's fault.